packages/react-divider/src/components/Divider/renderDivider.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import { DividerState } from './Divider.types';
+import { dividerShorthandProps } from './useDivider';
+
+/**
+ * Function that renders the final JSX of the component
+ */
+export const renderDivider = (state: DividerState) => {
+  const { slots, slotProps } = getSlots(state, dividerShorthandProps);
+  const { children, labelledById } = state;
+
+  return (
+    <slots.root {...slotProps.root} role="separator" aria-labelledby={labelledById}>
+      {children !== undefined && <slots.wrapper {...slotProps.wrapper} />}
+    </slots.root>
```

Comment:
Given how the `wrapper` slot is used, I wonder if `label` is a better name for it? (Feel free to leave it as-is if you disagree.)

Reply:
It is technically a wrapper as it is an accessibility wrapper for the content (labeling is is just the implementation)